src/javaharness/java/arcs/api/PECInnerPort.java

```diff
@@ -29,7 +29,5 @@ void HandleRemoveMultiple(
       PortableJson data,
       String particleId);
 
-  void Render(Particle particle, String slotName, PortableJson content);
-
-  // TODO: add more methods.
+  void Output(Particle particle, PortableJson content);
```

Comment:
method names always start with a lowercase letter.

Reply:
Did you fix this yet? only constructors in Java can be uppercase.

Reply:
yes, i pushed the fix, but forgot to comment, sorry: https://github.com/PolymerLabs/arcs/pull/3543/files#diff-ad507187aba8434905d04d3a87df45deR32